pkg/pipeline/pipeline_builder.go

```diff
@@ -0,0 +1,335 @@
+package pipeline
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/decode"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/encode"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/extract"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/ingest"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/transform"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/write"
+	"github.com/netobserv/gopipes/pkg/node"
```

Comment:
Do we have to include code review of `github.com/netobserv/gopipes/pkg/node` in this PR?

Reply:
Actually it is merged in its original repo. I'd maybe do it apart. Feel free to open a PR in gopipes with some suggestions, questions, or comments, and I can address them.